docs/code_coverage.md

```diff
@@ -2,11 +2,20 @@
 
 ## Prerequisites
 
-You need some additional Python modules to run the unit tests and linting.
+* Install all `TorchServe` prerequisites. For detailed steps refer [TorchServe installation documentation](../README.md#install-torchserve).
+
+* Install additional Python modules to run the unit tests and linting using following commands
+
+```bash
+cd <path_to_serve>
```

Comment:
Is there any reason for changing this from `cd serve` to `cd <path_to_serve>`

Reply:
User could be anywhere on the terminal, just making sure we are at the correct path before executing the next command.